splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceDateTimeFormatter.java

```diff
@@ -35,6 +35,17 @@
         TIME
     }
 
+    public static final String defaultDateFormatString = "yyyy-MM-dd";
+    public static final String defaultTimeFormatString = "HH:mm:ss";
+    public static final String defaultTimestampFormatString = "yyyy-MM-dd HH:mm:ss";
+
+    public static SpliceDateTimeFormatter defaultDateFormatter =
+        new SpliceDateTimeFormatter(null, FormatterType.DATE);
```

Comment:
What about a new constructor of `SpliceDateTimeFormatter` so we could avoid this `null`s? I'm thinking of `public static SpliceDateTimeFormatter SpliceDateTimeFormatter.of(FormatterType type)`.

Reply:
Sure.  Added as recommended.